build.gradle

```diff
@@ -133,6 +134,10 @@ dependencyCheck {
   }
 }
 
+ext {
+  set('springCloudVersion', "2021.0.5")
```

Comment:
Is there a particular reason for downgrading the version here?

Reply:
Yes, Spring Boot 2.7.7 is not compatible with Spring Cloud higher versions like 2022.0.0 etc for FeignClient libraries, application was failing on startup